packages/steveo/src/consumers/sqs.ts

```diff
@@ -66,15 +67,19 @@ class SqsRunner extends BaseRunner implements IRunner {
               await this.deleteMessage(c.topic, message);
             }
 
-            this.logger.debug('Start subscribe', c.topic, params);
             if (!task) {
               this.logger.error(`Unknown Task ${c.topic}`);
               return;
             }
 
-            await task.subscribe(params, runnerContext);
-            this.logger.debug('Completed subscribe', c.topic, params);
-            const completedContext = getContext(params);
+            const { context = null, ...data } = c.payload;
+            this.logger.info(
+              { context, data },
+              `Start Subscribe to ${task.name}`
+            );
+            await task.subscribe(data, context);
```

Comment:
I don't think this needs to change from passing params to the first argument. We don't know if there is anything using the context from the first argument, so if we don't pass it along it becomes a breaking change

Reply:
@sugendran I think it would also be the same for case for the existing runnerContext, we don't know if anything is using it so it might be worth to keep it

```
await task.subscribe(params, {...context, ...runnerContext});
```